SimPEG/EM/Analytics/FDEM.py

```diff
@@ -69,6 +69,16 @@ def MagneticDipoleWholeSpace(XYZ, srcLoc, sig, f, moment=1., orientation='X', mu
 
     """
 
+    if not isinstance(orientation, str):
```

Comment:
this code can later be improved by using Traitlets http://traitlets.readthedocs.io/en/stable/